lib/car.rb

```diff
@@ -0,0 +1,24 @@
+class Car
+  attr_reader :car, :make, :model, :monthly_payment,
+  :loan_length, :color
+  def initialize(make, model, loan_length)
+    @car = car
```

Comment:
No need to initialize the same class in this way; you don't use `@car` in this code anywhere. Only the things that are absolutely necessary for this class to work should be in the `#initialize` method.